draft-bbs-signatures.md

```diff
@@ -388,80 +390,77 @@ Procedure:
 
 ### Sign
 
-Sign computes a signature from SK, PK, over a vector of messages. This method
-describes deterministic signing. For threshold signing, XOF can be replaced
-with a PRF due to the insecurity of deterministic threshold signing.
+This operation computes a deterministic signature from a secret key (SK), over a header and a vector of messages.
 
 ```
-signature = Sign(SK, PK, (msg_1,..., msg_L), (H_1,..., H_L), header)
+signature = Sign(SK, header, (msg_1,..., msg_L), (H_1,..., H_L))
 
 Inputs:
 
-- msg_1,..., msg_L, octet strings. Messages to be signed.
-- H_1,..., H_L, points of G1. Generators used to sign the messages.
-- SK, a secret key output from KeyGen.
-- PK, a public key output from SkToPk.
-- header, an optional octet string containing context and application specific
-          information. If not supplied, it defaults to an empty string.
+- SK (REQUIRED), an octet string of the form outputted by the KeyGen operation.
+- header (OPTIONAL), an octet string containing context and application specific information. If not supplied, it defaults to an empty string.
+- msg_1,..., msg_L (OPTIONAL), a vector of octet strings. Application specific messages.
```

Comment:
Not really a big fun of defining the messages as "Application specific". It's definitely correct, however, it may raise the question of "what other messages are there??"

Reply:
Have removed for now, we can further refine this language later.